hack/mock-gen.sh

```diff
@@ -23,10 +23,6 @@ LICENSE_HEADER_PATH="./hack/boilerplate/boilerplate.go.txt"
 cd $GOPATH/src/github.com/koordinator-sh/koordinator
 
 # generates gomock files
-mockgen -source pkg/koordlet/statesinformer/states_informer.go \
```

Comment:
we should keep mockstatesinformer

Reply:
pkg/koordlet/statesinformer/states_informer.go **no such file**

Reply:
ops, it has been moved to pkg/koordlet/statesinformer/impl/states_informer.go